src/NuGet.Clients/NuGet.CommandLine/Commands/Pack/AssemblyMetadataExtractor.cs

```diff
@@ -9,7 +10,8 @@
 namespace NuGet.CommandLine
 {
     using NuGet.Packaging;
-    using Versioning;
+    using NuGet.Versioning;
```

Comment:
This is actually required, or else it starts picking up incorrect classes from other namespaces.  Like it was picking up NuGet.PackageBuilder instead of NuGet.Packaging.PackageBuilder.